docs/source/dev/buildoptions.rst

```diff
@@ -42,7 +42,7 @@ CMake Option                   Values          Description
 ``BUILD_SHARED_LIBS``          ON/**OFF**      Build the C++ API as shared library
 ``Boost_USE_STATIC_LIBS``      ON/**OFF**      Require static Boost library
 ``HDF5_USE_STATIC_LIBRARIES``  ON/**OFF**      Require static HDF5 library
```

Comment:
Presumably we'd want to do the same for HDF5?

Reply:
No, these vars are `FindPackagename.cmake` specific (legeacy) and not fully unified in the CMake community :)